app/src/main/java/com/github/featuredetectandroid/utils/PreferencesManager.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.featuredetectandroid.utils
+
+import android.content.Context
+import android.util.Log
+import androidx.core.content.edit
+import com.github.featuredetectandroid.MainActivity
+
+private const val TAG = "PreferencesManager"
+private const val KEY = "algorithm"
+private const val NONE = "None"
```

Comment:
Let's call these `ALGORITHM_KEY` and `ALGORITHM_DEFAULT` as we may extend this class to handle more keys

Reply:
fixed